data/recData/notify.js

```diff
@@ -0,0 +1,69 @@
+class Notify {
+  constructor() {
+    self.port.on('data', recs => {
+      this.removeOldBoxes();
+      this.createNewBoxes(recs);
+    });
+  }
+
+  //removes old recommendations, that may be for a different domain
+  removeOldBoxes() {
```

Comment:
Naming this function something like `clearRecommendations` would probably make it obvious enough that the comment isn't needed. Also, I'd avoid using the terms `new` and `old`  here, since the remover doesn't really distinguish by the age of the recommendation.